### ISSUE TITLE
RoboDomo

### ISSUE THREAD

**Issue by mschwartz** [mqtt-smarthome/mqtt-smarthome#24]
Hi, 

I am the author of RoboDomo, a Node.js and React based Home Automation system.  The back end is fully MQTT based - there are micro services to handle all sorts of devices, including Harmony Remote hub, SmartThings hub, LG TV, Sony TV, Nest thermostat, Autelis Pool Controller, and more.

The client software is React and Bootstrap based.  The client uses MQTT for all it’s state gathering and requesting of state changes.

The backend is dockerized so you can trivially bring up the various device integrations, and thus the MQTT micro services are available on dockerhub.

https://github.com/RoboDomo

Screenshots available here:

Https://github.com/RoboDomo/RoboDomo

**Comment by mschwartz** [comment]
I thought you might add the software to your list.

*shrug*

**Comment by hobbyquaker** [comment]
Hi! No offense :) Just like to keep the Issuetracker here for "real Issues"... As far as I saw RoboDomo does not follow the mqtt-smarthome architecture proposal (e.g. no +/status/# and +/set/# topics)? If I got that wrong I would be happy to merge a PR for the list here. If not I would suggest to add RoboDomo to https://github.com/hobbyquaker/awesome-mqtt - just send me a PR and I will merge.
Regards
